.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@
 - `yarn test:unit:all` should run without errors or warnings
 - `yarn serve` should run without errors or warnings
 - `yarn build` should run without errors or warnings
+- For API configuration changes: CDK tests added/updated in `backend/compact-connect/tests/unit/test_api.py`
```

Comment:
Home page pops up with search, but what if I just want to look at the list of providers? It's not obvious how to do that. (Was it an intended design choice to show the search modal on load?) From an accessibility POV, it's unclear how to close the modal and show the results, too. When I use voiceover, nothing obviously allows me to see results.

Reply:
It appears from the designs that an initial search is required. I think this may have stemmed from the lack of paging support and wanting to force users to search first rather than scanning pre-loaded records. It looks like that approach has carried into the hi-fi designs. If you think that's something we should discuss and possibly change, we could create a discussion ticket (or otherwise remember to bring up at our next touch point).

Reply:
Got it. In that case, why does the "search" button appear at all while the search modal is up? I can currently click it to close the modal, but it then says "no results" which is confusing. For the sake of this sprint's PR, can we do something simple like disable the "search" button when the search modal is active? That'll make it more intuitive and accessible for screen readers (and, I hope, is quick to implement.)

Reply:
@jlkravitz I can start working on something that addresses that, and is also maybe is in the direction of the upcoming hi-fi. I've created #242 

Which branch should I cut that from and merge to? There is already some significant work on `development` since then. For branch history sake it will probably be simplest to treat this as new work in the current sprint. Thoughts?